packages/webpack-config/src/webpack/base/module.js

```diff
@@ -1,24 +1,29 @@
-import merge from 'webpack-merge';
+import merge from "webpack-merge";
 
 // Settings
-import { moduleEntrySettings } from './settings/entry/module';
-import { moduleOutputSettings } from './settings/output/module';
-import { contextSettings } from './settings/context';
-import { resolveSettings } from './settings/resolve';
+import { moduleEntrySettings } from "./settings/entry/module";
+import { moduleOutputSettings } from "./settings/output/module";
+import { contextSettings } from "./settings/context";
+import { resolveSettings } from "./settings/resolve";
 
 // Tasks
-import { getJSLoader } from './tasks/jsLoading/getJSLoader';
-import { loadFonts } from './tasks/loadFonts'
-import { compileShadowCSS } from './tasks/compileShadowCSS';
-import { loadHandlebars } from './tasks/loadHandlebars';
+import { getJSLoader } from "./tasks/jsLoading/getJSLoader";
+import { loadFonts } from "./tasks/loadFonts";
+import { compileShadowCSS } from "./tasks/compileShadowCSS";
+import { loadHandlebars } from "./tasks/loadHandlebars";
+
+// CompileHTML
+import { compileHTML } from "./tasks/compileHTML";
+import { useHtmlCompiler } from "../../helpers/paths";
 
 export const defaultConfigModule = merge(
   moduleEntrySettings,
   moduleOutputSettings,
   contextSettings,
   resolveSettings,
-  getJSLoader('module'),
+  getJSLoader("module"),
   compileShadowCSS,
+  useHtmlCompiler ? compileHTML : {},
   loadFonts,
-  loadHandlebars,
-);
\ No newline at end of file
+  loadHandlebars
```

Comment:
I think we should use trailing commas to keep githistory clean

Reply:
As discussed to make this clean and in a way pv-standards are defined I will add pv-eslint rules.